src/core/iomgr/iomgr.h

```diff
@@ -37,11 +37,22 @@
 /* gRPC Callback definition */
 typedef void (*grpc_iomgr_cb_func)(void *arg, int success);
 
+typedef struct grpc_iomgr_cb {
+  grpc_iomgr_cb_func cb;
+  void *cb_arg;
+  int success;
+  int is_ext_managed;  /** is memory being managed externally? */
```

Comment:
Ack, I figured. The reason for its existence is in the pull req. description. Let's chat in person.